src/pint/simulation.py

```diff
@@ -22,7 +22,7 @@
 ]
 
 
-def zero_residuals(ts, model, maxiter=10, tolerance=None):
+def zero_residuals(ts, model, subtract_mean=True, maxiter=10, tolerance=None):
```

Comment:
Since these are not keyword-only arguments, someone might have been using these positionally and their code will break. Perhaps it's better to put the new argument at the end? Or if breaking their code is okay, make these arguments keyword-only?

Reply:
I used the order since more people would change `subtract_mean` compared to the others, although that isn't a big deal.  But I made them keyword-only as you suggested.